GoogleApisClient.sln

```diff
@@ -335,6 +335,12 @@ Global
 		{39E1EC7C-3C05-44E4-A19F-A61BF6EA9B11}.ReleaseSigned|x64.ActiveCfg = ReleaseSigned|Any CPU
 		{39E1EC7C-3C05-44E4-A19F-A61BF6EA9B11}.ReleaseSigned|x86.ActiveCfg = ReleaseSigned|x86
 		{39E1EC7C-3C05-44E4-A19F-A61BF6EA9B11}.ReleaseSigned|x86.Build.0 = ReleaseSigned|x86
+		{39E1EC7C-3C05-44E4-A19F-A61BF6EA9B11}.ReleaseTravis|Any CPU.ActiveCfg = ReleaseSigned|Any CPU

```

Comment:
Gah, no. The solution file shouldn't be there at all. Not sure how that happened. It should just be the project files and new code...